WcaOnRails/app/models/competition.rb

```diff
@@ -107,7 +108,7 @@ def delegates_must_be_delegates
     end
   end
 
-  def warnings
+  def warnings_for(user)
```

Comment:
Yeah... I think there are better ways to do `html_safe`, but what I was thinking doesn't work.